Source/WebCore/Modules/webaudio/AudioContext.cpp

```diff
@@ -187,6 +189,20 @@ double AudioContext::baseLatency()
     return static_cast<double>(destination().framesPerBuffer()) / sampleRate();
 }
 
+double AudioContext::outputLatency()
+{
+    lazyInitialize();
+
+    if (noiseInjectionPolicies().contains(NoiseInjectionPolicy::Minimal))
```

Comment:
I think you could go down this codepath in both cases (`Minimal` and `Enhanced`)

```suggestion
    if (noiseInjectionPolicies())
```